api/v1alpha1/envoygateway_types.go

```diff
@@ -91,6 +91,28 @@ type EnvoyGatewaySpec struct {
 	ExtensionAPIs *ExtensionAPISettings `json:"extensionApis,omitempty"`
 }
 
+type KubernetesClient struct {
+	// RateLimit defines the rate limit settings for the Kubernetes client.
+	RateLimit *KubernetesClientRateLimit `json:"rateLimit,omitempty"`
+}
+
+// KubernetesClientRateLimit defines the rate limit settings for the Kubernetes client.
+type KubernetesClientRateLimit struct {
+	// QPS defines the queries per second (QPS) limit for the Kubernetes client.
+	// If unspecified, defaults to 50. Min value must be greater than or equal to 1.
+	//
+	// +optional
+	// +kubebuilder:validation:Minimum=1
```

Comment:
adding the 50 and 100 default tags here will show up in the docs under the default coloumn